compat/src/render.js

```diff
@@ -227,3 +227,11 @@ export const __SECRET_INTERNALS_DO_NOT_USE_OR_YOU_WILL_BE_FIRED = {
 		}
 	}
 };
+
+options.beforeSetEventProperty = name => {
```

Comment:
rather than introducing a new event hook we could do something along the lines of

```js
if (onChange && onInput) {
  // combine them into one callback
}
```

This way we aren't creating more hooks

Reply:
I think that approach would be fine as long as we handle the error for the first callback executed.

<img width="1091" alt="image" src="https://user-images.githubusercontent.com/11670923/173222758-bd5151f8-569b-4f46-ac99-2c5d5b67a751.png">


I guess, we have different entries in the event queue when using different listeners  which are executed independently.

Reply:
>In relation to your issue, the issue states that you want actual onChange behavior when you tab, React just runs them both as onInput. Are we going for the latter?

@JoviDeCroock I included the React behavior in the issue. I think that would be the expected behavior

Reply:
I Implemented the new approach, thanks for the quick review!